diff --git a/node_modules/@typescript-eslint/utils/dist/ts-eslint/Rule.d.ts b/node_modules/@typescript-eslint/utils/dist/ts-eslint/Rule.d.ts
index 5ae7288..3d14a45 100644
--- a/node_modules/@typescript-eslint/utils/dist/ts-eslint/Rule.d.ts
+++ b/node_modules/@typescript-eslint/utils/dist/ts-eslint/Rule.d.ts
@@ -11,6 +11,10 @@ export interface RuleRecommendationAcrossConfigs<Options extends readonly unknow
     strict: Partial<Options>;
 }
 export interface RuleMetaDataDocs<Options extends readonly unknown[]> {
+    /**
+     * The category the rule falls under
+     */
+    category?: string;
     /**
      * Concise description of the rule
      */
@@ -20,7 +24,7 @@ export interface RuleMetaDataDocs<Options extends readonly unknown[]> {
      * Used by the build tools to generate the recommended and strict configs.
      * Exclude to not include it as a recommendation.
      */
-    recommended?: RuleRecommendation | RuleRecommendationAcrossConfigs<Options>;
+    recommended?: RuleRecommendation | RuleRecommendationAcrossConfigs<Options> | boolean;
     /**
      * The URL of the rule's docs
      */
diff --git a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
index 38d1f12..bf897a3 100644
--- a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
+++ b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
@@ -113,7 +113,7 @@ interface RunTests<MessageIds extends string, Options extends Readonly<unknown[]
     readonly invalid: readonly InvalidTestCase<MessageIds, Options>[];
 }
 interface RuleTesterConfig extends ClassicConfig.Config {
-    readonly parser: string;
+    readonly parser?: string;
     readonly parserOptions?: Readonly<ParserOptions>;
 }
 declare class RuleTesterBase {
