diff --git a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
index 38d1f12..bf897a3 100644
--- a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
+++ b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
@@ -113,7 +113,7 @@ interface RunTests<MessageIds extends string, Options extends Readonly<unknown[]
     readonly invalid: readonly InvalidTestCase<MessageIds, Options>[];
 }
 interface RuleTesterConfig extends ClassicConfig.Config {
-    readonly parser: string;
+    readonly parser?: string;
     readonly parserOptions?: Readonly<ParserOptions>;
 }
 declare class RuleTesterBase {
