diff --git a/node_modules/eslint-doc-generator/dist/lib/option-parsers.js b/node_modules/eslint-doc-generator/dist/lib/option-parsers.js
index 556ff0d..0361b02 100644
--- a/node_modules/eslint-doc-generator/dist/lib/option-parsers.js
+++ b/node_modules/eslint-doc-generator/dist/lib/option-parsers.js
@@ -27,9 +27,9 @@ export function parseConfigEmojiOptions(plugin, configEmoji) {
         if (plugin.configs?.[config] === undefined) {
             throw new Error(`Invalid configEmoji option: ${config} config not found.`);
         }
-        if (RESERVED_EMOJIS.includes(emoji)) {
-            throw new Error(`Cannot specify reserved emoji ${EMOJI_CONFIG_ERROR}.`);
-        }
+        // if (RESERVED_EMOJIS.includes(emoji)) {
+        //     throw new Error(`Cannot specify reserved emoji ${EMOJI_CONFIG_ERROR}.`);
+        // }
         return [{ config, emoji }];
     }) || [];
     // Add default emojis for the common configs for which the user hasn't already specified an emoji.
